_posts/en/newsletters/2019-08-07-newsletter.md

```diff
@@ -0,0 +1,149 @@
+---
+title: 'Bitcoin Optech Newsletter #58'
+permalink: /en/newsletters/2019/08/07/
+name: 2019-08-07-newsletter
+slug: 2019-08-07-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter announces the maintenance release of Bitcoin Core
+0.18.1, summarizes a discussion about BIP174 extensions, and notes a
+proposal for LN trampoline payments.  Our *bech32 sending support*
+section this week features a special case study contributed by BRD and
+our *notable changes* section highlights several possibly significant
```

Comment:
Are we planning on emphasizing the sections moving forward? Was not sure if this was intentional.

Reply:
No, this was a special case requested by @moneyball https://github.com/bitcoinops/bitcoinops.github.io/pull/176/files/07e854c6acc8aaf9ccbce9a7b16972d0ed040833#diff-ef4489117fbc2501b497cd6ec1feb572 (which I agree with).  The code changes section is only emphasized because it sounded weird to me to only emphasize the bech32 section (and because it really was a week with lots of changes).